src/Dialogs/Imap/ImapSavePage.vala

```diff
@@ -18,13 +18,36 @@
 *
 */
 
-public class OnlineAccounts.ImapDonePage : Gtk.Grid {
+public class OnlineAccounts.ImapSavePage : Gtk.Grid {
     public signal void back ();
     public signal void close ();
 
-    private Gtk.Stack alert_view_stack;
+    private Gtk.Stack stack;
+    private Gtk.Button back_button;
+    private Gtk.Button close_button;
+    private Granite.Widgets.AlertView error_alert_view;
+    private GLib.Cancellable? cancellable = null;
 
     construct {
+        var busy_label = new Gtk.Label (_("Configuring the mail account…"));
+
+        var busy_spinner = new Gtk.Spinner ();
+        busy_spinner.start ();
+
+        var busy_grid = new Gtk.Grid () {
+            column_spacing = 6
+        };
+        busy_grid.add (busy_label);
+        busy_grid.add (busy_spinner);
+
+        error_alert_view = new Granite.Widgets.AlertView (
+            _("Error"),
+            _("Unable to add mail account."),
```

Comment:
It looks like we always set the error description to whatever is passed in via `show_error`, so we should probably leave the description empty and use this string here for the title:
```suggestion
            _("Unable to add mail account"),
            ""
```